pkg/auth/dependency/hook/mutator_impl.go

```diff
@@ -104,12 +108,23 @@ func (mutator *mutatorImpl) Apply() error {
 		authInfo.DisabledMessage = ""
 		authInfo.DisabledExpiry = nil // never expire
 	}
-	if mutations.IsVerified != nil {
-		authInfo.Verified = *mutations.IsVerified
+	if mutations.IsManuallyVerified != nil {
+		authInfo.ManuallyVerified = *mutations.IsManuallyVerified
 	}
 	if mutations.VerifyInfo != nil {
 		authInfo.VerifyInfo = *mutations.VerifyInfo
 	}
+
+	if mutations.IsManuallyVerified != nil || mutations.VerifyInfo != nil {
```

Comment:
Seems we don't need to recompute the verified if user change the `IsManuallyVerified ` only? Maybe we should choose either,
1. We don't need this check and always recompute
1. Recompute only if `mutations.VerifyInfo != nil` ?

Reply:
Yes you're right, `Verified` flag in auth info does not depends on `ManuallyVerified`. I'll make it so that it is recomputed only if `VerifyInfo` is mutated.